packages/notifi-react-hooks/lib/hooks/useNotifiClient.ts

```diff
@@ -275,6 +276,7 @@ const useNotifiClient = (
   const [loading, setLoading] = useState<boolean>(false);
   const [isAuthenticated, setIsAuthenticated] = useState<boolean>(false);
   const [expiry, setExpiry] = useState<string | null>(null);
+  const [isTokenExpired, setisTokenExpired] = useState<boolean>(false);
```

Comment:
can't wait til we consolidate to using frontend-client in react-hooks too

Reply:
yeah, that's def something i want to do.